June 2026 newsletter (#1040)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2026-06-05-newsletter.md

- [Hidden REF 2026 is open](https://hidden-ref.org/hidden-ref-competition/), and they've added new categories including environmental sustainability. - Submission deadline is **2026-09-13**

---

### Events

<!-- Nothing scheduled but left as a place-holder should this be used as a template for next newsletter -->

<!-- ### RSE lunchtime seminars -->

<!-- Members of Sheffield RSE will be hosting a new small seminar series in the new year, with **catered lunches**! -->

<!-- You can sign up on [MyDevelopment]() (requires sign in to MUSE). -->

#### Upcoming Events in Sheffield

- **Last chance** - The [Bio-AI interest group](https://sheffield.ac.uk/machine-intelligence/community/interest-groups/bioscience-and-ai) is organising a sandpit discussion event on the topic of AI-powered biological image analysis on **1230-1430 07/07/2026, Diamond Workroom 3**.
  - The event is open to all and encourages participation from interested colleagues working in diverse fields (e.g. bioscience, computer science, medicine, geography, psychology, engineering etc.).
  - If you are interested in attending and/or presenting, please [sign up](https://forms.gle/CEDfdvKdPTj4u9mc9) by **8th June 2026**.

> - Tools for Data FAIRification - overcoming barriers in data discoverability and reuse - Online Seminar: **1300-1500 23 June 2026** -  [Register here](https://forms.gle/jthN9JDR6hUsQAUx8)  

Reviewer: ```suggestion
- Tools for Data FAIRification - overcoming barriers in data discoverability and reuse - Online Seminar: **1300-1500 23rd June 2026** -  [Register here](https://forms.gle/jthN9JDR6hUsQAUx8)  
```

---
layout: post
title: June 2026 Newsletter
editor: Dan Brady
slug: 2026-06-05-newsletter

> date: 2026-06-05T12:00:00Z+0100

Reviewer: This will have be published after `12:00:00Z` (but probably not an issue).

tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter June 2026

Welcome to the June 2026 newsletter for the research software community at The University of Sheffield, featuring
news, opportunities, events and training for you.

<!-- --- -->

### News

- **RSE Call for Proposals 2026 is live**  

>   6 months at 0.5 FTE RSE collaboration is available for four projects, two of which are funded by the Henry Royce Institute and will support projects from Materials Science. [Full details](https://rse.shef.ac.uk/collaboration/RSEtime/2026/) - Submission deadline is **1700 BST 10/07/2026**

Reviewer: [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) dates are consistent and avoid any ambiguity.

```suggestion
  6 months at 0.5 FTE RSE collaboration is available for four projects, two of which are funded by the Henry Royce Institute and will support projects from Materials Science. [Full details](https://rse.shef.ac.uk/collaboration/RSEtime/2026/) - Submission deadline is **2026/07/10 17:00 BST**
```

> - [Hidden REF 2026 is open](https://hidden-ref.org/hidden-ref-competition/), and they've added new categories including environmental sustainability.

Reviewer: ```suggestion
- [Hidden REF 2026 is open](https://hidden-ref.org/hidden-ref-competition/), and they've added new categories including environmental sustainability. - Submission deadline is **2026-09-13**
```

---

### Events

<!-- Nothing scheduled but left as a place-holder should this be used as a template for next newsletter -->

<!-- ### RSE lunchtime seminars -->

<!-- Members of Sheffield RSE will be hosting a new small seminar series in the new year, with **catered lunches**! -->

<!-- You can sign up on [MyDevelopment]() (requires sign in to MUSE). -->

#### Upcoming Events in Sheffield

> - **Last chance** - The [Bio-AI interest group](https://sheffield.ac.uk/machine-intelligence/community/interest-groups/bioscience-and-ai) is organising a sandpit discussion event on the topic of AI-powered biological image analysis on **1230-1430 07/07/2026, Diamond Workroom 3**.

Reviewer: ```suggestion
- **Last chance** - The [Bio-AI interest group](https://sheffield.ac.uk/machine-intelligence/community/interest-groups/bioscience-and-ai) is organising a sandpit discussion event on the topic of AI-powered biological image analysis on **2026/07/07 12:30-14:30, Diamond Workroom 3**.
```

Reviewer: ```suggestion
- **Last chance** - The [Bio-AI interest group](https://sheffield.ac.uk/machine-intelligence/community/interest-groups/bioscience-and-ai) is organising a sandpit discussion event on the topic of AI-powered biological image analysis on **1230-1430 7th July 2026, Diamond Workroom 3**.
```

  - The event is open to all and encourages participation from interested colleagues working in diverse fields (e.g. bioscience, computer science, medicine, geography, psychology, engineering etc.).

>   - If you are interested in attending and/or presenting, please [sign up](https://forms.gle/CEDfdvKdPTj4u9mc9) by **08/06/2026**.

Reviewer: ```suggestion
  - If you are interested in attending and/or presenting, please [sign up](https://forms.gle/CEDfdvKdPTj4u9mc9) by **2026/08/06**.
```

> - Tools for Data FAIRification - overcoming barriers in data discoverability and reuse - Online Seminar: **1300-1500 23/06/2026** -  [Register here](https://forms.gle/jthN9JDR6hUsQAUx8)  

Reviewer: ```suggestion
- Tools for Data FAIRification - overcoming barriers in data discoverability and reuse - Online Seminar: **2026/23/06 13:00-15:00** -  [Register here](https://forms.gle/jthN9JDR6hUsQAUx8)  
```

This seminar is part of Sheffield’s [Data Stewards Network](https://sheffield.ac.uk/library/research-data-management/data-steward-network) in collaboration with the [N8 Centre of Excellence in Computationally Intensive Research](https://n8cir.org.uk/).  
Talks include:
  - Streamlining Data Management Planning - Kryštof Komanec (Czech Technical University / DSW / FAIR Wizard)
  - Maximising Data Reusability - Matthew Nichols (UK Centre for Ecology & Hydrology)
  - Automated ReadMe Generation - Ali Abdurhman Kelil & Dag-Even Martinsen Torsøe (University of South-Eastern Norway)
  - Electronic Notebooks - Zuzanna Zagrodzka (University of Manchester)  

#### Upcoming External Events

- **Last chance** - [Agile Project Management Training Programme](https://research-it.manchester.ac.uk/events/agile-project-management-training-for-drtps/)  

> **Online, 0930-1630 19/06/2026** - Sign-up deadline: **10/06/2026**  

Reviewer: ```suggestion
**Online, 0930-1630 2026/19/06** - Sign-up deadline: **2026/10/06**
```

Designed for digital Research Technical Professionals (dRTPs) to help bridge the gap between traditional research and Agile delivery.  
Programme Highlights:
  - Targeted Learning: Covers Scrum, Kanban, and hybrid frameworks specifically for research environments.
  - Hands-on Experience: Includes simulated team projects and real-world application.
  - Fully Funded: Supported by the DisCouRSE Network+ for dRTPs across the N8 Research Partnership and UCL.

---

### Articles, Blogs, Papers, Pre-Prints, Podcasts/Videos & Books

#### Articles

- [Redefining Publishing : Practical pathways to Open Science](https://explore.plos.org/practical-pathways-to-open-science) - This report presents key findings from PLOS’ 18-month investigation into Open Science and publishing.

#### Blogs

- [Creating a fully open environment for research code and data - LSE Impact](https://blogs.lse.ac.uk/impactofsocialsciences/2024/09/11/creating-a-fully-open-environment-for-research-code-and-data/) -
Details how Nature Computational Science went about being fully compliant with open research and data principles

#### Papers

> - [Journal of Open Source Software: FAIRLinked: Data FAIRification Tools for Materials Data Science](https://joss.theoj.org/papers/10.21105/joss.09628)

Reviewer: Other papers don't list the journal they are published in...

```suggestion
- [FAIRLinked: Data FAIRification Tools for Materials Data Science](https://joss.theoj.org/papers/10.21105/joss.09628)
```

- [Peer code review in research software development: The research software engineer perspective](https://doi.org/10.1007/s10664-025-10785-0)

- [Skills for the curation of sensitive data](https://doi.org/10.5281/zenodo.19883038)

- [Sensitive data curation: evidence-based skills and training landscape for stakeholders](https://doi.org/10.5281/zenodo.19883313)

<!-- #### Pre-Prints -->
<!-- #### Podcasts & Videos -->

#### Books

- [Test-Driven Data Analysis](https://book.tdda.info/) - 

> The goal of this book is to reduce the frequency and severity of errors in all forms of data processing and analysis, from the simplest query to the most complex machine-learning data pipeline

Reviewer: ```suggestion
The goal of this book is to reduce the frequency and severity of errors in all forms of data processing and analysis, from the simplest query to the most complex machine-learning data pipeline.
```

#### Training Materials

> - [Managing Research Software Projects](https://third-bit.com/mrsp/)

Reviewer: ```suggestion
- [Managing Research Software Projects](https://third-bit.com/mrsp/) workshop material.
```

---

### Opportunities

> - UKRI are looking for new members to join the UKRI Advisory Group for Digital Research Infrastructure (AGD). Deadline 7 August 2026 (5pm). Full deatils [here](https://www.ukri.org/who-we-are/work-for-us/join-an-advisory-committee-panel-or-network/digital-research-infrastructure-advisory-group-member-vacancies/).

Reviewer: To be consistent in date/time formatting...

```suggestion
- UKRI are looking for new members to join the UKRI Advisory Group for Digital Research Infrastructure (AGD). Deadline **2026/08/07 17:00**. Full details [here](https://www.ukri.org/who-we-are/work-for-us/join-an-advisory-committee-panel-or-network/digital-research-infrastructure-advisory-group-member-vacancies/).
```

#### Surveys

> - [Exploring the usage of project management practices among digital research technical professionals (dRTPs) at the University of Sheffield](https://forms.gle/e5FWuWqTeye6Xg3g6) - This survey explores the usage of project management practices among individuals working at the Univerist of Sheffield who categorise themselves as digital research technical professionals (dRTPs) | **Deadline 10/07/2026**

Reviewer: Submission/deadline elsewhere isn't in bold.

```suggestion
- [Exploring the usage of project management practices among digital research technical professionals (dRTPs) at the University of Sheffield](https://forms.gle/e5FWuWqTeye6Xg3g6) - This survey explores the usage of project management practices among individuals working at the Univerist of Sheffield who categorise themselves as digital research technical professionals (dRTPs). Deadline **2026/10/07**
```

> - [Exploring how Research Technicians and Technology & Skills Specialists (RTPs) engage with open research, the challenges faced, and what support, training, and recognition are needed](https://www.qualtrics.manchester.ac.uk/jfe/form/SV_8CGJpNZnV6Ecwgm) - This survey focuses on how RTP roles contribute to strengthening research integrity, transparency, and reproducibility, and how these contributions are recognised and supported across institutions. We are also interested in understanding how the technician community is perceived, the challenges faced in engaging with open research, and what support, resources, or recognition could further enable RTPs to contribute to open and trustworthy science. | **Deadline 15/06/2026**

Reviewer: ```suggestion
- [Exploring how Research Technicians and Technology & Skills Specialists (RTPs) engage with open research, the challenges faced, and what support, training, and recognition are needed](https://www.qualtrics.manchester.ac.uk/jfe/form/SV_8CGJpNZnV6Ecwgm) - This survey focuses on how RTP roles contribute to strengthening research integrity, transparency, and reproducibility, and how these contributions are recognised and supported across institutions. We are also interested in understanding how the technician community is perceived, the challenges faced in engaging with open research, and what support, resources, or recognition could further enable RTPs to contribute to open and trustworthy science. Deadline **2026/15/06**
```

> - [Impact of Generative AI on Software Development, Application and Licensing Models](https://forms.gle/gbEiGSsnN9mZXk2L9) - Originally inspired by research software engineers from within materials modelling communities this survey aims to build our understanding and capture usage and adoption of Generative AI in the domain of Research Software Engineering

Reviewer: Shame there is no obvious deadline for submission on this, only that results will be published "_by end of July 2026_".
